geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/converters/FastConverter.scala

```diff
@@ -0,0 +1,84 @@
+/***********************************************************************
+ * Copyright (c) 2013-2019 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.utils.geotools.converters
+
+import java.util.concurrent.ConcurrentHashMap
+
+import com.typesafe.scalalogging.StrictLogging
+import org.geotools.factory.GeoTools
+import org.geotools.util.{Converter, Converters}
+
+import scala.util.control.NonFatal
+
+/**
+  * Replacement for geotools `Converters`, which caches the converters for each pair of (from -> to) instead of
```

Comment:
The GeoTools converters recreates the class each time?

Reply:
yeah... https://github.com/geotools/geotools/blob/master/modules/library/metadata/src/main/java/org/geotools/util/Converters.java#L148